[DOCU-2466][DOCU-2463] Konnect Rename + Reorder (#4412)
Kong/docs.konghq.com: app/konnect/dev-portal/access-and-approval/manage-devs.md

no_version: true
content-type: reference
---

> Within the {% konnect_icon dev-portal %} **Dev Portal** section there are three ways to manage access and authorization settings for the Dev Portal: 

Reviewer: ```suggestion
In the {% konnect_icon dev-portal %} **Dev Portal** section, there are three ways to manage access and authorization settings for the Dev Portal: 
```

* The **Access Requests** page lists developer registration and application registration requests.

> * The **Developers** page list of developers who have requested access to the Dev Portal, along with the developer's current status and associated applications.

Reviewer: ```suggestion
* The **Developers** page lists developers who have requested access to the Dev Portal, along with the developer's current status and associated applications.
```

* The **Applications** page lists applications and their statuses.

To allow automatic approval of developer registration requests,
enable [auto approve](/konnect/dev-portal/access-and-approval/auto-approve-devs-apps).

## Access Requests page

> The **Access Requests** page is available by clicking {% konnect_icon dev-portal %} **Dev Portal**, from the {{site.konnect_short_name}} navigation menu. On the **Access Requests** page, you can manage registration requests for developers using the **Developers** tab, and application requests using the **Applications** tab. There are three options for managing incoming requests: **Approve**, **Reject**, and **Delete**. When auto-approve is disabled all registration requests must be manually approved from the **Access Requests** page. 

Reviewer: ```suggestion
The **Access Requests** page is available by clicking {% konnect_icon dev-portal %} **Dev Portal**, from the {{site.konnect_short_name}} navigation menu. On the **Access Requests** page, you can manage registration requests for developers using the **Developers** tab, and application requests using the **Applications** tab. When auto-approve is disabled, all registration requests must be manually approved from the **Access Requests** page. 
```
I think this content could be cut down a little bit because this is explained in the task sections that follow this one.

> ### Manage Developer access request {#approve-dev-reg}

Reviewer: ```suggestion
### Manage developer access requests {#approve-dev-reg}
```

> All pending developer requests are displayed in the **Developers** tab within the **Access Requests** section..

Reviewer: ```suggestion
All pending developer requests are displayed in the **Developers** tab in the **Access Requests** section.
```

> To approve or reject an access requests, the **Developers** page, approve a developer's registration request:

Reviewer: ```suggestion
To approve or reject an access requests, go to the **Developers** page:
```

> 1. Find the developer request you want to approve.

Reviewer: ```suggestion
1. Find the developer request you want to view.
```

2. Click the {% konnect_icon cogwheel %} icon and choose

>    **Approve**, **Reject** or **Delete** from the context menu.

Reviewer: ```suggestion
   **Approve**, **Reject**, or **Delete** from the menu.
```

The three options available are defined below: 

* **Approve** - The developer access request is approved. The developer is able to log in to the Dev Portal. The developer is now visible from the **Developers** page in {{site.konnect_short_name}}. 
* **Reject** - The developer access request is rejected. Rejected developers are removed from the requests list. The developer is unable to submit another registration request.
* **Delete** - The developer access request is deleted. A developer can submit another registration request. To prevent a developer from submitting requests, you must [revoke](#revoke-dev-access) access.

## Developers page

> From this page you can manage users who have already been approved and are actively using the Dev Portal. 

Reviewer: ```suggestion
From this page, you can manage users who have already been approved and are actively using the Dev Portal. 
```

### Revoke access {#revoke-dev-access}

Revoking access prevents a developer from signing up for access again with the same
email address, unless the revoked request is deleted. Reasons for revoking a developer can include
violations of your organization's policies or other nefarious reasons.

To revoke a developer's access, from the **Developers** page, follow these steps:

1. Find the developer whose status you want to change.
2. Click the {% konnect_icon cogwheel %} icon and choose **Revoke** from the
   context menu.

> This action can be undone, by clicking the {% konnect_icon cogwheel %} and choosing **Approve**.

Reviewer: ```suggestion
This action can be undone by clicking the {% konnect_icon cogwheel %} and choosing **Approve**.
```
